packages/mobile/src/screens/edit-track-screen/screens/ListSelectionScreen.tsx

```diff
@@ -17,11 +16,12 @@ export type ListSelectionData = { label: string; value: string }
 export type ListSelectionProps = {
   screenTitle: string
   icon: ComponentType<SvgProps>
-  searchText: string
   data: ListSelectionData[]
   renderItem: ListRenderItem<ListSelectionData>
-  onChange: (value: Nullable<string>) => void
+  onChange: (value: any) => void
```

Comment:
need to be able to set other types of values

Reply:
Maybe use a type parameter to ListSelectionProps?